src/PluginRegistrationDialog.cpp

```diff
@@ -700,8 +671,7 @@ void PluginRegistrationDialog::SetState(int i, bool toggle, bool state)
    {
       item->state = state;
    }
-   
-   if (mFilter == ID_ShowNew && item->state != STATE_New)
```

Comment:
I don't understand why it's correct to remove the first condition.  Can you explain?